CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## vNext
 - ...
 
+## v0.24.4
```

Comment:
It should be `0.25.0`. It is not a hotfix. It is a new release.

Reply:
Actually, it was planned as a bug fix, not a hotfix or some new functionality. A hotfix is used when we need to fix something urgent in PROD and it's not the case. Here I did a bug fix for an old functionality but it's not urgent. This is why I updated the patch version.

Reply:
@DmitriyMolch 
Anything we are releasing from `dev` is new major or minor releases. Please, update the version to `0.25.0`

Reply:
In Jira this task was designed as a bug and not like a new feature. Only features can have minor version updates.

Reply:
ok